regions/io/crtf/read.py

```diff
@@ -482,10 +482,20 @@ def parse_coordinate(string_rep):
             return u.Quantity(string_rep[:-3], u.dimensionless_unscaled)
         if 'h' in string_rep or 'rad' in string_rep:
             return Angle(string_rep)
+
+        # CRTF format apparently implicitly defines
+        # 18:20:30.12
+        # strings as hour units, and
+        # 10.11.54.69
+        # as degrees.
+        # I can't find this definition in documentation anywhere, though.
         if len(string_rep.split('.')) >= 3:
             string_rep = string_rep.replace('.', ':', 2)
+            unit = u.deg
+        elif string_rep.count(":") == 2:
+            unit = u.hour
```

Comment:
```suggestion
        unit = u.deg
        if len(string_rep.split('.')) >= 3:
            string_rep = string_rep.replace('.', ':', 2)
        elif string_rep.count(":") == 2:
            unit = u.hour
```
Should there be a default `unit`, e.g. shall 10.119 be parsed as `10.119 * u.deg`?
Thought that was causing the test to fail, but actually it is just not raising the right exception now, expecting input like '18h12m24s' to raise `CRTFRegionParserError`, getting `undefined variable 'unit'` instead.